source/voting.rst

```diff
@@ -0,0 +1,96 @@
+.. _voting:
+
+############
+Voting Items
+############
+
+Decisions about the future of the project are made through discussion with all
+members of the community. All non-sensitive project management discussion takes
+place on the project developers’ mailing list, the issue tracker, and in the
+project meetings mentioned above. Occasionally, sensitive discussion occurs on
+a private list, but the project defaults to open meetings.
+
+The yt-project uses a “consensus seeking” process for making large,
+project-changing decisions. The
+group tries to find a resolution that has no open objections among project
+members. At any point during the discussion, any project member can call for
+a vote, which will conclude two weeks from the call for the vote. Any vote must
+be backed by a YTEP. If no option can gather two thirds of the votes cast, the
```

Comment:
What does it mean for a vote to be backed by a YTEP?

Reply:
I think we might already do this, but if we're making a large (like a big API change) change to the project and a vote is called, there must be a YTEP associated with that vote. I think we already default to creating YTEPs in changes of that magnitude, but it ensures that votes can be cast either in the YTEP itself or elsewhere. We could force the vote to happen only in one place (like the YTEP) though.

Reply:
I think we need to make the voting process concrete. Is this +1 on a github issue or messages to a mailing list or...?

Reply:
Ok I've hopefully clarified this. Feel free to bring up something if you feel it could be improved.